modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/src/main/resources/META-INF/resources/partials/form_elements.jsp

```diff
@@ -49,7 +53,7 @@
 		</tr>
 		<tr>
 			<td><clay:checkbox indeterminate="<%= true %>" label="My Input" name="name" showLabel="<%= false %>" /></td>
-			<td>Checkbox Variable for multiple selection</td>
+			<td>Checkboxes can utilize the <em>:indeterminate</em> pseudo class when manually set via JavaScript (there is no available HTML attribute for specifying it).</td>
```

Comment:
Who is this message for? :)

Based on this, we're basically providing a broken API that does nothing. Instead, same as we do with other attributes that require JS, we should detect it and then in presence of `indeterminate`, trigger the hydration of the component to get the component in the proper state.

Reply:
> we're basically providing a broken API that does nothing

Yep, I'll investigate further and work on hydration, wasn't really sure about this one, now you've clarified it, thanks!